Py/DICOMSegmentationPlugin.py

```diff
@@ -143,54 +143,67 @@ def load(self,loadable):
       (success,labelNode) = slicer.util.loadLabelVolume(labelFileName, returnNode=True)
 
       # Initialize color and terminology from .info file
-      # Format of the .info file:
+      # See SEG2NRRD.cxx for how it's written.
+      # Format of the .info file (no leading spaces, modifiers are optionally written if present):

```

Comment:
One round of unwrinkling would be useful as there's more over head on my side to get the minor change in. I've updated the file on this branch to init the strings, now looking at how to get that into your branch (feel free to cherry-pick it):
https://github.com/naucoin/Reporting/commit/35a789c6504e8c4ddcef46b0904a08e723b5ae4d